sycl/doc/extensions/experimental/sycl_ext_matrix/sycl_ext_oneapi_matrix.asciidoc

```diff
@@ -258,6 +304,13 @@ void joint_matrix_store(Group g,
    const joint_matrix<Group, T, use::accumulator, Rows, Cols, layout::dynamic> &res,
    multi_ptr<T, Space, IsDecorated> dest, size_t stride, layout Layout);
 
+template <typename Group, typename T1, typename T2, size_t Rows, size_t Cols,
+          typename PropertyListT>
+void joint_matrix_store(Group g,
+   const joint_matrix<Group, T1, use::accumulator, Rows, Cols, layout::dynamic> &res,
+   ext::oneapi::experimental::annotated_ptr<T2, PropertyListT> dest,
+   size_t stride, layout Layout);
```

Comment:
```suggestion
template <typename Group, typename T, size_t Rows, size_t Cols,
          typename PropertyListT>
void joint_matrix_store(Group g,
   const joint_matrix<Group, T, use::accumulator, Rows, Cols, layout::dynamic> &res,
   ext::oneapi::experimental::annotated_ptr<T, PropertyListT> dest,
   size_t stride, layout Layout);
```

The other overload of `joint_matrix_store` has only one `T` template parameter (not two), so I presume you want the same here.

Note, however, in another PR I suggested changing `joint_matrix_store` to have two template parameters `T1` and `T2`.  So, I do think it makes sense for *both* overloads to have these two template parameters.  However, I think you  should make that change in a separate PR, which changes both overloads.

Reply:
You are right. Let's add T1 and T2 in the other PR https://github.com/intel/llvm/pull/12276
Whichever PR gets merged first, the other one will follow.